### ISSUE TITLE
Text2image and image2image

### ISSUE THREAD

**Issue by AutumnQ10086** [comfyanonymous/ComfyUI#1889]
How to combine the part of text2image and the part of image2image? Seems that I can't implement both function in ome workflow.

**Comment by Th3Rom3** [comment]
What exactly are you trying to do?
You can always wire the image output of a sampler into a second sampler within the same workflow.

**Comment by AutumnQ10086** [comment]
Yes I want to wire the image output of a sampler into a second sampler within the same workflow, but "save image" have only one input port but no output port, what should I do?

**Comment by ltdrdata** [comment]
![image](https://github.com/comfyanonymous/ComfyUI/assets/128333288/877bba61-7574-4e16-b7c3-d311e51ae806)

**Comment by AutumnQ10086** [comment]
Thank you!

**Comment by AutumnQ10086** [comment]
Thank you! That helps a lot.

Dr.Lt.Data ***@***.***> 于2023年11月5日周日 10:10写道：
